source.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export LIBSODIUM_CLIENT_DIR=~/keystone/keystone-demo/libsodium_builds/libsodium_client/src/libsodium
```

Comment:
These all have to be relative to `pwd`, ex:
```
export LIBSODIUM_CLIENT_DIR=$(pwd)/libsodium_builds/libsodium_client/src/libsodium
export LIBSODIUM_DIR=$(pwd)/libsodium_builds/libsodium_server/src/libsodium
export KEYSTONE_SDK_DIR=$(pwd)/../sdk
```

Additionally we need to set `KEYSTONE_DIR`, and this should probably just use that instead.

Reply:
Oh I completely forgot that I still had those in here. I will fix that since it can be useful to use.

Reply:
I just commented out the bottom one. Also I did not want to do just ../sdk since I did not want to assume where the sdk was and I agree that we need to just set the `KEYSTONE_DIR` but I think we will need to make a PR in the Keystone repo for that.